python/ray/experimental/state/common.py

```diff
@@ -522,7 +522,17 @@ class TaskState(StateSchema):
     #: The runtime environment information for the task.
     runtime_env_info: str = state_column(detail=True, filterable=False)
     #: The parent task id.
-    parent_task_id: str = state_column(filterable=True)
+    parent_task_id: str = state_column(detail=True, filterable=True)
+    #: The list of events of the given task.
+    #: Refer to src/ray/protobuf/common.proto for a detailed explanation of the state
+    #: breakdowns and typical state transition flow.
+    events: List[dict] = state_column(detail=True, filterable=False)
+    #: The list of profile events of the given task.
+    profile_events: List[dict] = state_column(detail=True, filterable=False)
```

Comment:
nit: should we call it `status_events`?  Or just call `profile_events` `profiling_data`?

Or if this is good enough? I feel right now, it seems like `events` and `profile_events` might be a bit overlapping thus not the best naming.

Reply:
Hmm good point. I prefer events + profile_data in this case